setup.py

```diff
@@ -55,6 +55,8 @@ def run(self):
 
 install_requires = [
     "alembic>=0.6.6",
+    "amqp>=1.4.5,<2.0",
+    "anyjson>=0.3.3",
```

Comment:
I was installing pu on my local machine and when I was trying to run `inveniomanage bower` I got error that anyjson is missing.